python/interpret_community/widget/explanation_dashboard.py

```diff
@@ -19,32 +19,32 @@
     raise RuntimeError("Error: gevent package is missing, please run 'conda install gevent' or"
                        "'pip install gevent' or 'pip install interpret-community[visualization]'")
 
-"""Explanation Dashboard Class.
+class ExplanationDashboard:
 
```

Comment:
This looks like a lot of changes, but I just moved the docstring from above the "class ExplanationDashboard" to below. That way, the text shows up in the sphinx html docs.